site/docs/guidelines/localization.mdx

```diff
@@ -4,6 +4,51 @@ route: /guidelines/localization
 menu: Guidelines
 ---
 
-# Localisation
+import LargeParagraph from "../../src/components/LargeParagraph";
 
-_Coming soon!_
+
+# Guidelines for localisation
+<LargeParagraph>
+    Localising services for different languages makes City of Helsinki services more approachable and easy to understand for wider audiences.
+</LargeParagraph>
+
+Here are some recommendations for localising content and navigation:
+
+- **The default language of Helsinki digital services is Finnish (FI).**
+  - It is recommended that Helsinki digital services follow Finnish date and time locale even if the user is using some other language. See the [HDS data format guidelines](https://hds.hel.fi/guidelines/data-formats "HDS data format guidelines") for more information.
+- **Content should also be provided at least in Swedish (SV) and English (EN), if possible.**
+  - Finnish (FI), Swedish (SV) and English (EN) languages should be treated as equal in content and language menus. For these languages it is recommended to provide full content.
+  - For English (EN) locale, British English spelling should be used.
+  - For Swedish (SV) locale, the Helsingfors version of the framed logo should be used. Other languages use the default Helsinki framed logo.
+- **Additional languages**, sign language and plain language versions of content can be added depending on the targeted user group of the service. The content provided in additional languages can be a subset or a summary of the main content.
+
+
+- **Let the user select their preferred language.**
+  - Do not auto-localise content or menus unless the user has picked a default language for them self. 
+  - For logged in users the language selection can be a stored in user information.
+  - For non logged in users, the selection made in language picker can be stored with a cookie.
+- **When the user switches languages, the page context should be preserved.**
+  - When user switches language on a sub page, the corresponding page should be shown, instead of redirecting the user to front page.
+  - If a localised version of the content is not available, it is recommended to inform the user and provide possible next actions, instead of redirecting to front page.
+- **The preferred method for switching languages is with the language picker action in [HDS Navigation component](https://hds.hel.fi/components/navigation "HDS Navigation component documentation").**
+  - The available languages can also be listed as links in the navigation bar instead of dropdown, if the services navigation is simple and there are less than three available languages.
```

Comment:
> The available languages can also be listed as links in the navigation bar instead of dropdown, if the services navigation is simple and there are less than three available languages.

Tällaista ohjetta ei taida itse navigaation dokumentaatiossa lukea. Pitäisiköhän tämä lisätä sinnekin?

Reply:
Tää tuli oikeastaan siitä keskustelusta FI/SV/EN tasaveroisuudesta ja erillisestä kieli/saavutettavuuspalkki-ajatuksesta. Ja kukkuussahan näin on nyt tehty. Ehkä lisään tämän myös navin dokumentaatioon. Vai mahdollistaako nav-komponentti ylipäätään tämän kaltaisen kikkailun?